things/service.go

```diff
@@ -147,6 +147,30 @@ type thingsService struct {
 	ulidProvider mainflux.IDProvider
 }
 
+// provideUUID: Generates UUID or takes external ID if supplied.
+func (ts *thingsService) provideUUID(extID string) (id string, err error) {
+	if(extID == "") {
+		id, err = ts.idProvider.ID()
+	} else {
+		id, err = extID, nil
+	}
+
+	return id, err
+
+/*
+	if(extID != "") {
+		id, err = ts.idProvider.FromString(extID)
+		if(err != nil) {
+			id, err = ts.idProvider.ID()
+		}
+	} else {
+		id, err = ts.idProvider.ID()
+	}
+
+	return id, err
+*/
```

Comment:
Remove dead code

Reply:
That was my mistake.  Let me remove that immediately.